Makefile.Common

```diff
@@ -99,7 +99,7 @@ checklicense:
 .PHONY: checklinks
 checklinks:
 	command -v $(MDLINKCHECK) >/dev/null 2>&1 || { echo >&2 "$(MDLINKCHECK) not installed. Run 'npm install -g markdown-link-check'"; exit 1; }
-	find . -name \*.md -print0 | xargs -0 -n1 \
+	find . -name \*.md -print0 | sort | uniq | xargs -0 -n1 \
```

Comment:
Is this change related or is it an improvement to the checklinks target?

Reply:
it's small improvement for getting list in the same order